move/gas_service/sources/gas_service.move

```diff
@@ -67,8 +70,13 @@ public struct GasCollected<phantom T> has copy, drop {
 fun init(ctx: &mut TxContext) {
```

Comment:
`init` can't be called on an upgraded package, right? just on the first deployment iirc?

Reply:
yes